projects/1.16/assets/relics-of-gaming/rogblades/lang/zh_cn.json

```
@@ -0,0 +1,42 @@
+{
+   "itemGroup.rog_mod":"拔刀剑：游戏遗物",
+
+  "rogblades.name": "拔刀剑：游戏遗物",
+  "rogblades.desc": "来自游戏中的武器。",
+
+  "item.slashblade.ex.beastlord": "百兽剑王",
+  "item.slashblade.ex.beastlord.desc": "游戏《尼尔：机械纪元》中的武器。",
+  "item.slashblade.ex.beastlord.material": "百兽剑王拵",

```

```suggestion
  "item.slashblade.ex.beastlord.material": "刀组件「百兽剑王」",
```
库内拔刀剑2本体的blade component一律按日语译为“刀组件”，要求附属也统一。请全文通查并修改